design/PROCESSES.md

```diff
@@ -0,0 +1,15 @@
+# UML Tool
+Everyone must use the same UML tool because they are usually not compatible with one another.
+As of yet, we did not decide on a specific UML tool to use.
+
+Must-have features:
+
+- Cross-platform use (Windows and Linux)
```

Comment:
I'm personally fine with both, but didn't we agree to use only Linux.

Reply:
Currently Mike is still using Windows.
If I remember correctly Joe is also using Windows.
If we want to start modelling as soon as possible we should pick a tool that can be used across platforms.
Otherwise we would have to wait until they can set up a Linux machine.
(This requirement is actually rather weak. So far I have only seen one tool that was exclusive to one platform.)